### PR TITLE
Create SECURITY.md

### PR DESCRIPTION
**Pull request by bast0006**

### CODE REVIEW

**Comment by bast0006** [comment]
Boilerplate. Add more if you think we need it.

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+the current dev state is supported security wise. Please do not remain on old 
+versions long, as that may make contributing difficult.
+
+| Version | Supported          |
+| ------- | ------------------ |
+| < 1.0   | :white_check_mark: |
+
+## Reporting a Vulnerability
+
+Please report vulnerabilities here on discord using the
```
Comment (dawnofmidnight) [inline SECURITY.md:15]: Shouldn't it say "here on GitHub"?

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+| Version | Supported          |
+| ------- | ------------------ |
+| < 1.0   | :white_check_mark: |
+
+## Reporting a Vulnerability
+
+Please report vulnerabilities here on discord using the 
+"report security vulnerability" flow. As this is active development, expect
+a response within a week or two. If it has gone longer than that, or you can 
+see this and are sure the report-this flow is nonfunctional, join our [discord](https://discord.gg/3f7jBUXdxw)
```
Comment (dawnofmidnight) [inline SECURITY.md:18]: Make sure to capitalize Discors

**Review by dawnofmidnight — DISMISSED** [review]

**Review by onerandomusername — CHANGES_REQUESTED** [review]
What is the "report security vulnerability" flow?

**Comment by bast0006** [comment]
Oh for the love of

**Review by onerandomusername — APPROVED** [review]
LGTM

**Pull request merged**